src/components/personkort/PersonkortHeader/PersonkortHeader.tsx

```diff
@@ -28,27 +32,33 @@ const StyledFnr = styled.div`
 const PersonkortHeader = () => {
   const navbruker = useNavBrukerData();
   const personident = useValgtPersonident();
-  const { hasGjentakendeSykefravar } = useOppfolgingstilfellePersonQuery();
+  const { data: maksDato } = useMaksdatoQuery();
 
   return (
     <>
       <div className="flex personkortHeader__info">
         <KjonnIkon personident={personident} />
         <div>
-          <h3 className="flex items-center">
-            {`${navbruker.navn} (${hentBrukersAlderFraFnr(personident)} år)`}
-            {hasGjentakendeSykefravar && (
-              <Tooltip content={"Gjentatt sykefravær"}>
-                <ArrowsCirclepathIcon className="ml-2" />
-              </Tooltip>
-            )}
-          </h3>
+          <NavnHeader navnSykmeldt={navbruker.navn} personident={personident} />
 
           <StyledFnr>
             {formaterFnr(personident)}
             <CopyButton message={texts.copied} value={personident} />
           </StyledFnr>
-          <SyketilfelleSummary />
+
+          <div className="flex flex-row gap-3 items-center">
+            <TilfellePeriod />
+            <Varighet />
+            <Diagnosekode />
+          </div>
```

Comment:
Mulig vi kan bruke `HStack` i stedet for div her: https://aksel.nav.no/komponenter/primitives/hstack

Reply:
Funka bra det! 🤌